tnp/templates/consent/profile.html

```diff
@@ -0,0 +1,395 @@
+		                            
+		                            <h6> 
+		                            	<b>Permanent Address</b>
+		                            </h6>
+		                            {{ personaldetail.permanent_address }}
+		                        </div>
+		                    </div>
+
+		                    <div class="row">
+		                        <div class="input-field col s8 offset-s2 m4 offset-m1">
+		                        	
+		                            <h6> 
+		                            	<b>Current Residence City</b>
+		                            </h6>
+		                            {{ personaldetail.current_residence_city }}
+		                        </div>
+		                        <div class="input-field col s8 offset-s2 m4 offset-m2">
+		                            
+		                            <h6>
+		                            	<b>Permanent Residence City</b>
+		                            </h6>
+		                            {{ personaldetail.permanent_residence_city }}
+		                        </div>
+		                    </div>
+
+		                    <div class="row">
+		                        <div class="input-field col s8 offset-s2 m4 offset-m1">
+		                        	
+		                            <h6> 
+		                            	<b>Current Residence State</b>
+		                            </h6>
+		                            {{ personaldetail.current_residence_state }}
+		                        </div>
+		                        <div class="input-field col s8 offset-s2 m4 offset-m2">
+		                            
+		                            <h6> 
+		                            	<b>Permanent Residence State</b>
+		                            </h6>
+		                            {{ personaldetail.permanent_residence_state }}
+		                        </div>
+		                    </div>
+	                    </div>
+	                    <div class="card grey lighten-5 grey-text text-darken-2 z-depth-2 hoverable">
+	                    	<div class="card-content grey darken-2 grey-text text-lighten-5 z-depth-2">
+	                      		<span class="card-title"> Educational Details </span>
+	                    	</div>
+
+	                    	<div class="section"> </div>
+
+	                    	<div class="row">
+		                        <div class="input-field col s6 offset-s2 m3 offset-m1">
+		                        	
+		                            <h6><b>
+		                            	SSC
+		                            </b>
+		                            </h6>
+		                            {{ education_detail.ssc }}
+		                        </div>
+		                        <div class="input-field col s2 offset-s1 m1">
+		                            
+		                            <h6> 
+		                            	<b>Ssc Result Type</b>
+		                            </h6>
+		                            {{ education_detail.ssc_result_type }}
```

Comment:
Instead of explicitly displaying Result Type as `PERCENTAGE` show the symbol. 
`SSC: 89 %`

Use `{{ education_detail.get_ssc_result_type_display }}` to show % sign